cmd/dlt/accountroles/cmd.go

```diff
@@ -182,8 +194,8 @@ func setDeleteRoles(isClassicFlagSet bool, isHostedCPFlagSet bool) (bool, bool)
 	return isClassicFlagSet, isHostedCPFlagSet
 }
 
-func deleteAccountRoles(r *rosa.Runtime, env string, prefix string, clusters []*cmv1.Cluster, mode string,
-	hostedCP bool) error {
+func deleteAccountRoles(r *rosa.Runtime, cmd *cobra.Command, env string, prefix string, clusters []*cmv1.Cluster,
+	mode string, hostedCP bool) error {
```

Comment:
If you are going to supply the cmd I suppose you could remove the mode param :thinking:

Reply:
Ah actually after looking at it, I don't think we should, `mode` is a var scoped to the `run` function which can be changed after the flag itself

Reply:
I could include the `cmd.Flag(...).Changed` value though instead